src/components/molecules/customers-groups-summary/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { sortBy } from "lodash"
+
+import { CustomerGroup } from "@medusajs/medusa"
+
+/**
+ * Customers Associated Groups props
+ */
+interface P {
+  groups: CustomerGroup[]
+}
+
+/**
+ * Render a summary of groups to which the customer belongs
+ */
+function CustomersGroupsSummary(props: P) {
+  const groups = sortBy(props.groups, "name")
+  if (!groups.length) return null
+
+  const left = groups.length - 1
+  const leadName = groups[0].name
```

Comment:
nit: just a suggestion - would it make sense to add a `limit` prop that defines "how many" groups will be displayed in the lead?

And then maybe rename this molecule to something that signifies that this is also usable for other areas not just customer groups. Maybe "LabelSummary", "EllipsisList", "EllipsisLabels" or something like that :D

Reply:
Let's leave it as is for now and keep a refactor in mind if we come across the need for a similar component in the future! :)